src/order-basket/drug-search.ts

```diff
@@ -69,7 +69,7 @@ function* explodeDrugResultWithCommonMedicationData(drug: Drug, encounterUuid: s
             asNeeded: false,
             asNeededCondition: '',
             startDate: new Date(),
-            duration: 0,
+            duration: null,
```

Comment:
Yeah that `null`;)

Reply:
In hindsight, that was kind of obvious 😆